src/ERC721ContractMetadata.sol

```diff
@@ -176,4 +188,80 @@ contract ERC721ContractMetadata is
     function _baseURI() internal view virtual override returns (string memory) {
         return _tokenBaseURI;
     }
+
+    /**
+     * @notice Sets the address to receive royalties.
+     *
+     * @param newWallet The new wallet address.
+     */
+    function setRoyaltyAddress(address newWallet) external onlyOwner {
+        // Set the new royalty address.
+        _royaltyAddress = newWallet;
+
+        // Emit an event with the royalty address update.
+        emit RoyaltyAddressUpdated(newWallet);
+    }
+
+    /**
+     * @notice Sets the royalty basis points out of 10_000.
+     *
+     * @param newBps The value as an integer (e.g. 500 for 5%)
+     */
+    function setRoyaltyBasisPoints(uint256 newBps) external onlyOwner {
+        // Set the new royalty percent.
+        _royaltyBps = newBps;
```

Comment:
should add a check that bps is `<= 10_000`

Reply:
added in https://github.com/ProjectOpenSea/seadrop/pull/21/commits/2c3670c9540e6158f3f722eb46d3519cc8b716d5